.env.example

```diff
@@ -25,6 +25,15 @@ DB_PASSWORD=secret
 # Should CDash automatically remove old builds?
 #AUTOREMOVE_BUILDS=true
 
+# How many builds should CDash try to remove per iteration?
+# Consider tweaking this value if you notice the autoremove queries
+# are slow to execute.
+#AUTOREMOVE_BUILDS_BATCH_SIZE=10
+
+# Skip the autoremove builds step if more than this number of submissions
+# are waiting to be parsed.
+#AUTOREMOVE_BUILDS_SKIP_THRESHOLD=100
```

Comment:
Instead of a new configuration option, what do you think about using [queue priorities](https://laravel.com/docs/11.x/queues#queue-priorities) to automatically prioritize submissions if such jobs exist?  That way, the queuing system will automatically handle cleanup tasks whenever submission activity is low.

Reply:
Sounds like a good idea to me!